nailgun/entities.py

```diff
@@ -4992,8 +5010,8 @@ def path(self, which=None):
             'bulk/destroy',
         ):
             return f'{super().path(which="base")}/{which}'
-        elif which in ('upload_facts',):
-            return f'{super().path(which="base")}/facts'
+        elif which in ('upload_facts', 'bootc_images'):
+            return f'{super().path(which="base")}/{which}'
```

Comment:
I couldn't find `upload_facts` endpoint in the apidoc, is this change intentional?

Would moving `boot_images` to line 5010 do the same works since it returns the same path (R5012 vs R5014)